test/table.ts

```diff
@@ -2707,6 +2733,36 @@ describe('Bigtable/Table', () => {
           done();
         });
       });
+
+      it('should send attempt header', done => {
+        const error = new Error('retryable') as ServiceError;
+        error.code = 14; // Unavailable
```

Comment:
In the future, I think there is an enum that we should use instead of `14`, but not super critical.

Reply:
agreed, but thats out of scope for this PR